skimage/color/colorconv.py

```diff
@@ -140,6 +140,9 @@ def _prepare_colorarray(arr, force_copy=False, *, channel_axis=-1):
     """
     arr = np.asanyarray(arr)
 
+    if arr.shape == (3,):
+        arr = arr.reshape((1, 3))
```

Comment:
Hmm, I think this might be more subtle than this because we need to account for `channel_axis`. How about

```suggestion
    if arr.ndim == 1:
        expanded_axis = {-1: 0, 0: -1}[channel_axis]
        arr = np.expand_dims(arr, expanded_axis)
```